ci-operator/step-registry/cucushift/hypershift-extended/etcd-encryption/azure/health-check/cucushift-hypershift-extended-etcd-encryption-azure-health-check-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/hypershift-extended/etcd-encryption/azure/health-check/cucushift-hypershift-extended-etcd-encryption-azure-health-check-ref.yaml",
```

Comment:
If I remember right, per yesterday's discussion, you are going move the check to `ci-operator/step-registry/cucushift/hypershift-extended/health-check/etcd-encryption`, we changed that idea?

Reply:
Actually here I am not strictly controlling hypershift steps folder architecture, but just suggest to avoid using "health-check" to avoid easily being confused with another `health-check` step - `cucushift-hypershift-extended-health-check`.

Reply:
Updated to etcd-encryption/health-check/azure as discussed in DM.